scss/detail.scss

```diff
@@ -52,3 +53,15 @@
   width: 540px;
   border-bottom: 4px solid $earls-green;
 }
+
+.book-badge {
```

Comment:
Use after or before pseudo-classes for this

Reply:
Done